src/delegate/delegate.ios.ts

```diff
@@ -259,10 +258,35 @@ export class TwilioAppDelegate extends UIResponder
       }
   }
 
-  reportIncomingCall(from: String, uuid: any) {
+  reportIncomingCall(from: String, uuid: any, callInvite : TVOCallInvite) {
+    // let customParameters: NSDictionary <string,string> = callInvite.customParameters;
+    
+    // if(customParameters) {
+    //   console.debug("Custom Parameters Detected:",customParameters);
+    // }
+
+    console.log('richard:',callInvite.customParameters);
```

Comment:
could you please remove the comment code and this `console.log`

Reply:
Oops left that during debugging, i have removed this one 💯